### PR TITLE
Ft create contact us page  181364551

### PR DESCRIPTION
**Pull request by RACHID-WEBDEV**
#### What does this PR do?
Create the Contact us Page
#### Description of Task to be completed?
- Add `Textarea component` into the component folder` which can be used for several other pages/components
- Create the `contact` page inside `pages` folder

#### How should this be manually tested?
- Clone the Repository
- Run `yarn Install` in your terminal to install required dependencies
-  Run `yarn dev` to start the local development server on `localhost:3000/contact

### CODE REVIEW

File: components/form/textarea.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+};
+Textarea.propTypes = {
+  className: PropTypes.string,
+  label: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  placeholder: PropTypes.string
+};
+
+Textarea.defaultProps = {
+  label: null,
```
Comment (andela-hpopoola) [inline components/form/textarea.js:39]: `className` is missing here.

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. I left a comment but you should feel free to merge after fixing the comment. Thank you.

**Comment by andela-hpopoola** [comment]
Please note that the Pull Request Title `Ft create contact us page 181364551` didn't follow the naming convention that we discussed the last time. Also, you could have added a screenshot of the contact us page in the PR description. Thank you.

**Comment by RACHID-WEBDEV** [comment]
> Please note that the Pull Request Title `Ft create contact us page 181364551` didn't follow the naming convention that we discussed the last time. Also, you could have added a screenshot of the contact us page in the PR description. Thank you.

Okay  noted

**Pull request merged**